lib/config/config.go

```diff
@@ -20,8 +20,8 @@ const (
 	defaultFlushTimeSeconds = 10
 	defaultFlushSizeKb      = 25 * 1024 // 25 mb
 
-	flushIntervalSecondsStart = 5
-	flushIntervalSecondsEnd   = 6 * 60 * 60
+	FlushIntervalSecondsStart = 5
```

Comment:
I made these public so I can import them in our API server.

Reply:
It'd be a little clearer to call these `FlushIntervalSecondsMin` and `FlushIntervalSecondsMax`

Reply:
Sgtm

Reply:
sgtm